docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  db:
+    image: "postgres:latest"
+    container_name: "lotus_flare_db"
+    environment:
+      - POSTGRES_USER=lotus_flare
+      - POSTGRES_PASSWORD=lotus_flare
```

Comment:
It's not a valid approach to put credentials into config and post that config on github :) Environment variables may be used for that.

Reply:
Sure, i know about that. I did because i was in hurry